group_vars/all.yml

```diff
@@ -274,3 +276,6 @@ osd_objectstore: bluestore
 #introspection_timeout:
 foreman_url: https://foreman.example.com
 manila_enabled: false
+
+#image path for overcloud nodes
+remote_image_path: https://sf.redhat.com
```

Comment:
Do we still need this remote_image_path? I guess this was mainly because of some issues with using VLANs in overcloud networking and because of some issues with boot while using bios boot mode in older images. I think those issues might be fixed in the latest puddle now, so is this still required?

Reply:
Yes, I have tested it with the latest puddle and the issue with overcloud images still doesn't seem to fixed and during the provisioning it enters into the grub rescue mode